dashboards-observability/common/constants/explorer.ts

```diff
@@ -280,3 +280,8 @@ export const DATA_CONFIG_HINTS_INFO = {
   [BREAKDOWNS]:
     "Defines how each series is broken down. Breakdowns are 'by' clauses that subdivide the existing series.",
 };
+
+export const HEATMAP_DATA_CONFIG_HINTS_INFO = {
+  [AGGREGATIONS]: 'only one Series is allowed',
```

Comment:
Ideally what we can do here is to have shared info and custom field for different visualizations if they have additional/different content to show for info icon. but all visualizations where it uses dimensions/series/breakdowns should use above general text for explanations. for example, if heatmap has such restrictions, then you can concatenate string to form maybe something like `${sharedInfo}. ${heatmapCustomInfo}`.

Reply:
@mengweieric, updated the constant as below, it will get concatenated with actual series and dimensions info as suggested. and please suggest if changes are needed for text.
```
export const HEATMAP_DATA_CONFIG_HINTS_INFO = {
  [AGGREGATIONS]: `${DATA_CONFIG_HINTS_INFO[AGGREGATIONS]} and max one series is allowed`,
  [GROUPBY]: `${DATA_CONFIG_HINTS_INFO[GROUPBY]} and max two dimensions are allowed`,
};
```